packages/common/src/log.ts

```diff
@@ -86,7 +84,10 @@ function trace(...args) {
  * @param {string/number} level - log level in string or number
  * @returns {number/string} - level in number or string
  */
-function convertLogLevel(level) {
+// @ts-ignore
+export function convertLogLevel(level) {
+  warn('@cumulus/common/log.convertLogLevel() is deprecated after version 1.23.2 and will be removed in a future release.');
```

Comment:
Is this directly using `warn` in order to avoid a potential infinite loop via the `deprecate` function?

Reply:
Correct. The `util` module (which contains `deprecate`) includes `log`, so I didn't want `log` to then include `util`.